src/types/types.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod';
+// Zod Schema Definition
+export const CarpoolDataSchema = z.object({
+    destination: z.string(),
+    date_time: z.string(),
```

Comment:
should be coerced to a date object, https://stackoverflow.com/questions/75157299/how-can-zod-date-type-accept-iso-date-strings